### DISCUSSION TITLE
Change keyboard layout non-programatically

### DISCUSSION THREAD

**Discussion by ThatOneCalculator** [CuarzoSoftware/Louvre#35] [Ideas]

**Comment by ehopperdietzel** [comment]
??

**Comment by ThatOneCalculator** [comment]
Sorry for not putting a description (I was a bit tired), but pressing the <kbd>-</kbd> key on my standard american layout qwerty keyboard produces a `'` character when in a `louvre-views` graphical session, which doesn't occur on any other compositor or desktop environment.

**Comment by ehopperdietzel** [comment]
Ahhh, I see 😆. Yes, all the examples automatically load the Latin American keyboard by default (cause I am Chilean 🇨🇱, hehe). Since these are just examples, I didn't include an option to modify that through configuration. However, you can programmatically change it, as explained [here](https://cuarzosoftware.github.io/Louvre/md_md_tutorial_04.html).

By the way, does SRM 0.5.2 fix your previous issue?

**Comment by ThatOneCalculator** [comment]
I haven't had the chance to try SRM, I'll probably wait until the Arch package is updated.

**Comment by ehopperdietzel** [comment]
Just letting you know, the Arch SRM and Louvre packages are updated. I'm also thinking of adding a configuration file to modify the keymap in the examples for the next release.
